metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/ExtensionServiceImpl.java

```diff
@@ -528,20 +527,26 @@ private void rollBackPatternsFromHdfs(InstallContext context) throws Exception{
   private void writeExtensionConfiguration(InstallContext context) throws Exception {
     ParserExtensionConfig config = new ParserExtensionConfig();
     config.setParserExtensionParserName(context.extensionParserNames);
-    config.setExtensionsBundleID(context.bundleID.get());
-    config.setExtensionsBundleVersion(context.bundleVersion.get());
+    config.setExtensionIdentifier(context.extensionIdentifier.get());
+    config.setExtensionBundleID(context.bundleID.get());
+    config.setExtensionBundleVersion(context.bundleVersion.get());
     config.setExtensionBundleName(context.bundleName.get());
     config.setExtensionAssemblyName(context.extensionPackageName.get());
     config.setDefaultParserConfigs(context.defaultParserConfigs.get());
-    config.setDefaultEnrichementConfigs(context.defaultEnrichmentConfigs.get());
+    config.setDefaultEnrichmentConfigs(context.defaultEnrichmentConfigs.get());
     config.setDefaultIndexingConfigs(context.defaultIndexingConfigs.get());
     if(context.defaultElasticSearchTemplates.isPresent()) {
       config.setDefaultElasticSearchTemplates(context.defaultElasticSearchTemplates.get());
     }
-    ConfigurationsUtils.writeParserExtensionConfigToZookeeper(context.extensionPackageName.get(),config.toJSON().getBytes(), client);
+    ConfigurationsUtils.writeParserExtensionConfigToZookeeper(context.extensionIdentifier.get(),config.toJSON().getBytes(), client);
```

Comment:
What is the logic for changing package name to identifier?

Reply:
While working with the config, the fact that I had picked a field to use as the identifier, but was also something else seemed confusing.  Making the identifier explicit, and also making that field what it should be ( the assemblyName/packageName ) without truncating it seems to me to make it more understandable.